src/main.rs

```diff
@@ -183,9 +184,9 @@ async fn main() -> anyhow::Result<()> {
         };
 
         tracing::debug!(target: LOG_TARGET, "state {:?}", state);
-
         let (election_result, round) = state.complete();
 
+        prometheus::record_election(&election_result);
```

Comment:
dq: shouldn't this happen after sucessful insertion into db?

Reply:
I would say that it doesn't matter both are metrics for the same thing i.e, written as prometheus metric or in the db.